jsx/App/App.jsx

```diff
@@ -10,6 +10,9 @@ export function App() {
     <LocaleProvider>
         <Header />
         <footer>
+        <small> 
+        <span translate="yes">This website is powered by <a href='https://github.com/BrownCLPS/LingView/' target="_blank" rel="noopener noreferrer">Lingview</a> © 2022</span>
```

Comment:
I think we should put this text as a translatable by creating an item for this entry in LocaleConstants.jsx (jsx/App/locale/). Please follow the existing examples on the page, eg. Header.jsx, using `<TranslatableText dictionary={someText} />`. The main goal here is to create 3 text entries for the same text, one in English, one in Spanish, and one in French, and when the user switches the language for the page the text will be translated accordingly.